config/database.yml

```diff
@@ -2,24 +2,26 @@ default: &default
   adapter: postgresql
   encoding: unicode
   user: postgres
-  password: postgres
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 10 } %>
 
 development:
   <<: *default
+  password: postgres
   host: localhost
   database: development
 
 test:
   <<: *default
   #The host for the database is database in compose networking in ci, locally you have
   #to set the DATABASE_HOST env variable to localhost
+  password: postgres
   host: <%= ENV.fetch("DATABASE_HOST") { "database" } %>
   database: test
 
 production:
   <<: *default
-  host: production_database_address
-  database: workspace_production
-  username: workspace
-  password: <%= ENV['WORKSPACE_DATABASE_PASSWORD'] %>
+  host: <%= ENV['DATABASE_HOST'] %>
+  database: <%= ENV['DATABASE_NAME'] %>
+  user: <%= ENV['DATABASE_USER'] %>
+  password: <%= ENV['DATABASE_PASSWORD'] %>
```

Comment:
FYI: all of this can be overwritten with a single `DATABASE_URL` ENV (https://edgeguides.rubyonrails.org/configuring.html#configuring-a-database)